views/elements/course-filters.php

```diff
@@ -0,0 +1,196 @@
+							<option value="<?php echo esc_attr( $v['value'] ); ?>">
+								<?php echo esc_html( $v['option'] ); ?>
+							</option>
+						<?php endforeach; ?>
+					</select>
+				</div>
+				<button class="tutor-btn tutor-btn-outline-primary" id="tutor-admin-bulk-action-btn" data-tutor-modal-target="tutor-bulk-confirm-popup">
+					<?php esc_html_e( 'Apply', 'tutor' ); ?>
+				</button>
+			</div>
+		</form>
+		<?php endif; ?>
+		<?php
+		$search_query  = Input::get( 'search', '' );
+		$current_order = Input::get( 'order', 'DESC' );
+		$order_link    = add_query_arg( 'order', 'ASC' === $current_order ? 'DESC' : 'ASC', tutor()->current_url );
+
+		$current_page = Input::get( 'page', '' );
+		$sub_page     = Input::get( 'sub_page', '' );
+		$current_tab  = Input::get( 'tab', '' );
+		if ( '' === $sub_page && '' !== $current_tab ) {
+			$sub_page = $current_tab;
+		}
+
+		$url = '';
+		if ( '' === $sub_page && '' === $current_tab ) {
+			$url = "?page=$current_page";
+		} elseif ( '' === $current_tab ) {
+			$url = "?page=$current_page&sub_page=$sub_page";
+		} else {
+			$url = "?page=$current_page&tab=$current_tab";
+		}
+
+		$filters_count = count(
+			array_filter(
+				$data['filters'],
+				function( $filter ) {
+					$value = Input::get( $filter['field_name'] );
+					return null !== $value && '' !== $value;
+				}
+			)
+		);
+		?>
+
+		<div class="tutor-wp-dashboard-filter-right">
+			<div class="tutor-d-flex tutor-flex-wrap tutor-align-center tutor-justify-end tutor-gap-1 tutor-ml-16">
+				<?php if ( isset( $data['filters'] ) ) : ?>
+				<div class="tutor-wp-dashboard-filters tutor-dropdown-parent">
+					<button type="button" class="tutor-wp-dashboard-filters-button <?php echo esc_attr( $filters_count > 0 ? 'active' : '' ); ?>" action-tutor-dropdown="toggle">
+						<i class="tutor-icon-slider-horizontal"></i>
+						<span class="tutor-fs-6 tutor-color-secondary"><?php esc_attr_e( 'Filters', 'tutor' ); ?></span>
+						<?php if ( $filters_count > 0 ) : ?>
+						<span class="tutor-wp-dashboard-filters-line"></span>
+						<span class="tutor-wp-dashboard-filters-count"><?php echo esc_html( $filters_count ); ?></span>
+						<?php endif; ?>
+					</button>
+
+					<div class="tutor-dropdown" data-tutor-dropdown-persistent>
+						<div class="tutor-d-flex tutor-justify-between tutor-mb-16">
+							<span class="tutor-fs-6 tutor-fw-medium"><?php esc_html_e( 'Filters', 'tutor' ); ?></span>
+							<button class="tutor-btn tutor-btn-ghost" data-tutor-dropdown-close>
+								<i class="tutor-icon-times"></i>
+							</button>
+						</div>
+
+						<div class="tutor-d-flex tutor-flex-column tutor-gap-12px">
+							<?php foreach ( $data['filters'] as $filter ) : ?>
+								<?php if ( 'date' === $filter['field_type'] ) : ?>
+									<div class="tutor-wp-dashboard-filters-item">
+										<?php if ( isset( $filter['label'] ) && isset( $filter['show_label'] ) && true === $filter['show_label'] ) : ?>
+											<label class="tutor-form-label">
```

Comment:
I think a type check is not required here, we can use
`! empty($filter['show_label'])` instead of this : `isset(  ) && true === $filter['show_label']`